### DISCUSSION TITLE
Multiple Choice and GoTo

### DISCUSSION THREAD

**Discussion by Jabberwocky23** [SilentVoid13/Templater#1091] [Help] [answered]
I have 2 problem that I can not get solved:

1. i am looking for a possibility to select multiple answers in the query when creating the template. For example: Select Tags.
Here is a list of all tags with the possibility to select multiple.

2. is there a possible after confirming the last prompt a whole specific page is called and displayed. Instead of the note just created.

For any help I would be very grateful

**Comment by Zachatoo** [comment]
I wrote up a few ways you can do a "multi-select" suggester [here](https://zachyoung.dev/snippets/templater#multi-select-suggester). I'm not sure what you want to do with the tags, this should be enough to get started.

```js
<%*
let tags = Object.keys(app.metadataCache.getTags());
const selectedTags = [];
while (true) {
  const selectedTag = await tp.system.suggester(tag => tag, tags);
  if (!selectedTag) {
    break;
  }
  selectedTags.push(selectedTag);
  tags = tags.filter(tag => tag !== selectedTag);
}
if (selectedTags.length > 0) {
  // Do something with selected tags
}
_%>
```

For your second question, you could open a note in a new tab like [this](https://zachyoung.dev/snippets/templater#opening-files-in-new-tabs). Though that probably isn't quite what you want.

**Reply by Jabberwocky23** [reply]
Hi Zachatoo - super many thanks for your support. i will try it out right away.

With the second problem I mean:

When you create a note by template query -.
so "Enter the title", "Which project", "Which tag" , and then press return on the last entry, the created note appears in a tab.

I would like that the note that was just created does not appear - but another note - in my case the dashboard.

**Reply by Jabberwocky23** [reply]
This is what I have tried:

`
<%*
let items = ["Value1", " Value2", " Value3", " Value4", " Value5", " Value6", " Value7" ];
const selectedItems = [];
while (true) {
  const selectedItem = await tp.system.suggester(item => item, items);
  if (!selectedItem) {
    break;
  }
  selectedItems.push(selectedItem);
  items = items.filter(item => item !== selectedItem);

}%>


<%*
let tags = Object.keys(app.metadataCache.getTags());
const selectedTags = [];
while (true) {
  const selectedTag = await tp.system.suggester(tag => tag, tags);
  if (!selectedTag) {
    break;
  }
  selectedTags.push(selectedTag);
  tags = tags.filter(tag => tag !== selectedTag);
}
if (selectedTags.length > 0) {
  // Do something with selected tags
}
_%>
`

title:  <%* tR += title %>
tags:  todo, <%* tR += tags %>
created: <% tp.date.now("dddd Do MMMM YYYY HH:mm") %>
last modified: <% tp.file.last_modified_date("dddd Do MMMM YYYY HH:mm") %>
alias: <%* tR += title %>
priority:  <%* tR += farbe %>
projekt: <% tp.system.suggester(["Footage", "NFTs", "Print", "Website", "YouTube"], ["Footage", "NFTs", "Print", "Website", "YouTube"], false,  "Wähle Projektzugehörigkeit:") %>
test:  <%* tR += selectedItems %>


i have only two tags selected. The result ist this: 
One problem is the # which must not appear here - otherwise the tag will not be recognized

title:  sb brgbr gbrbrgb
tags:  todo, #todo,#projekt,#knowhow,#obsidian,#midjourney,#deutsch,#footage,#website,#print,#dashboard
created: Donnerstag 20. April 2023 09:17
last modified: Donnerstag 20. April 2023 09:17
alias: sb brgbr gbrbrgb
priority:  Mittel
projekt: Footage
test:   Value3, Value5, Value7

**Reply by Jabberwocky23** [reply, marked as answer]
Problem 1 has been solved: since I only want to provide pre-selected, fixed tags for selection, your snippet with the values helps me

Problem 2 remains at the moment

**Reply by Jabberwocky23** [reply]
Problem 2 is also solved

in the PlugIn quickadd you can specify under options that the note is NOT called after creation.

That's what I wanted - Still thanks for your help
